test/UserService.Mappers.UnitTests/ResponsesMappers/UserResponseMapperTests.cs

```diff
@@ -35,6 +36,7 @@ public void SetUp()
         {
             userResponseMapper = new UserResponseMapper();
 
```

Comment:
If in mapper comes request editUserRequest with CreatedAt = null. Where is test-case?

Reply:
How it can be? If request doesn’t have this field, it is sets automatically